enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +274,21 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Dynamic plugins will need to contain their own localization files and serve them
+per requested. All dynamic plugins must use a single localization namespace
+per language, named after the namespace, e.g. for `kubevirt-plugin` the filename
+would be `kubevirt-plugin.json`. Localization files need to be served under
+`/locales/{language}/{namespace}.json` path.
+All dynamic plugins are expected to use `plugin__` namespace prefix, e.g.
+`plugin__knative-plugin` or `plugin__kubevirt-plugin`. The request for the dynamic
+plugin locales will be proxied by console to plugin's `Pod`, using it's `Service`.
+Example request path for `console-demo-plugin` locales will look like:
+`/locales/resource.json?lng=en&ns=plugin__console-demo-plugin'`.
+This request will be proxied to:
+`<service-name>.<service-namespace>.svc.cluster.local:<service-port>/locales/{language}/{namespace}.json`
```

Comment:
```suggestion
`<service-name>.<service-namespace>.svc.cluster.local:<service-port>/locales/{language}/plugin__console-demo-demo.json`
```

Reply:
Im a bit confused here based on https://github.com/openshift/enhancements/pull/815#discussion_r662469872. shall the `-plugin` be part of the namespace or not ?

Reply:
Only if the `ConsolePlugin` resource name has `-plugin` in it. In this case, the plugin name is `console-demo-plugin` (as stated above in line 287), but for most plugins I'd expect `-plugin` *not* to be in the name.

I think it would be better to use a real example here like `kubevirt` instead of `console-demo-plugin`.